src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -24,6 +24,8 @@
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 use Symfony\Component\Routing\Route;
 use Symfony\Component\Routing\RouteCollection;
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
```

Comment:
var-dumper is not a mandatory dep of fwb, can we do without?

Reply:
i took it from here https://github.com/symfony/symfony/blob/e00d2f14fbaf947ad585216d811d5444e3217a36/src/Symfony/Component/Form/Console/Descriptor/TextDescriptor.php#L18 :) Basically we require vardumper to display complex values, but maybe we can do a fallback? maybe @yceruto has a suggestion